### ISSUE TITLE
Release v6.1.2 broken

### ISSUE THREAD

**Issue by Niek** [Pomax/bezierjs#193]
The recent release v6.1.2 is broken. Here's a quick test:
```bash
$ npm ls bezier-js
└── bezier-js@6.1.0
$ node -e "console.log(require('bezier-js'))"
{ Bezier: [class Bezier] }
# all good, now let's upgrade to v6.1.2
$ npm i bezier-js@6.1.2
$ node -e "console.log(require('bezier-js'))"   
{}
```
I assume it has something to do with the new esbuild.

**Comment by Pomax** [comment]
Which version of Node?

**Comment by Pomax** [comment]
changed the esbuild args and test with node 16, should be fixed now in v6.1.3

**Comment by Niek** [comment]
I can confirm this is fixed in v6.1.3, thanks!
